### PR TITLE
GCW-2335 channel setup

### PR DESCRIPTION
**Pull request by abhinav098**
Hello team this is a child branch for GCW-2335-basic-setup-required-for-chat, contains changes for channel creation and updating store.

### CODE REVIEW

File: app/services/push_service.rb [hunk 1]
```diff
@@ -1,13 +1,17 @@
 class PushService
   def send_update_store(channel, app_name, data)
-    channel = Channel.add_app_name_suffix(channel, ADMIN_APP) if app_name == ADMIN_APP
+    if [ADMIN_APP, BROWSE_APP, STOCK_APP].include?(app_name)
```
Comment (steveyken) [inline app/services/push_service.rb]: Check the code in Channel.rb but I don't think you need the `if` statement here. It's already taken care of.

File: app/services/push_service.rb [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+      channel = Channel.add_app_name_suffix(channel, app_name)
+    end
     SocketioSendJob.perform_later(channel, "update_store", data.to_json, true)
   end
 
   def send_notification(channel, app_name, data)
     data[:message] = ActionView::Base.full_sanitizer.sanitize(data[:message])
     data[:date] = Time.now.to_json.tr('"','')
-    channel = Channel.add_app_name_suffix(channel, ADMIN_APP) if app_name == ADMIN_APP
+    if [ADMIN_APP, BROWSE_APP, STOCK_APP].include?(app_name)
```
Comment (steveyken) [inline app/services/push_service.rb]: Same as above, you should just be able to send `Channel.add_app_name_suffix(channel, app_name)` as it takes care of not adding a prefix for donor app.

Comment (abhinav098) [reply app/services/push_service.rb]: sure @steveyken thanks for pointing out, I'll check it and get back to you.

File: app/services/push_service.rb (hunk 1 above)
Comment (abhinav098) [reply app/services/push_service.rb]: Yeah, I've made the changes. Thanks @steveyken 
here's the commit link https://github.com/crossroads/api.goodcity/pull/647/commits/868e249bd2344523814c6429b7e46d6dc1714da8

File: app/services/push_service.rb (hunk 2 above)
Comment (abhinav098) [reply app/services/push_service.rb]: I've made the changes. Here's the commit link [868e249](https://github.com/crossroads/api.goodcity/commit/868e249bd2344523814c6429b7e46d6dc1714da8)

**Pull request merged**